winch/codegen/src/codegen/context.rs

```diff
@@ -250,7 +289,7 @@ impl<'a> CodeGenContext<'a> {
     // we could effectively ignore that range;
     // only focusing on the range that contains
     // spillable values.
-    fn spill<M: MacroAssembler>(
+    fn spill_callback<M: MacroAssembler>(
```

Comment:
The naming here is a little confusing -- I would have expected this method to take a callback (closure) and call it for something, but I guess the idea is that this is actually meant to be called from within a callback, where we have an explicit borrow of `self.regalloc` and `self.frame`?

Maybe `spill_impl` instead?

Reply:
Yeah `spill_impl` makes sense to me.